### PR TITLE
rpmspec: add page

### PR DESCRIPTION
**Pull request by mfrw**
Signed-off-by: Muhammad Falak R Wani <falakreyaz@gmail.com>

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

Closes: #6778

### CODE REVIEW

File: pages/linux/rpmspec.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# rpmspec
+
+> Query a RPM spec file.
+> More information: <https://manned.org/rpmspec>.
+
+- List binary packages which would be generated from the rpm spec file:
```
Comment (navarroaxel) [inline pages/linux/rpmspec.md:6]: Should we use `specification`? :thinking: 
```suggestion
- List binary packages which would be generated from the rpm specification file:
```

File: pages/linux/rpmspec.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Get summary information for single binary packages generated from rpm spec file:
+
+`rpmspec --query --queryformat {{"%{name}: %{summary}\n"}} {{rpm.spec}}`
+
+- Get the source package which would be generated from the rpm spec file:
+
+`rpmspec --query --srpm {{rpm.spec}}`
+
+- Prase the rpm spec file to `stdout`:
```
Comment (navarroaxel) [inline pages/linux/rpmspec.md:18]:
```suggestion
- Parse the rpm spec file to `stdout`:
```

File: pages/linux/rpmspec.md (hunk 1 above)
Comment (mfrw) [reply pages/linux/rpmspec.md:6]: Its by default called the `spec` file .. and `.spec` is the extension.
I am pretty sure that would be short for `specification` but all the docs always mention `spec`

File: pages/linux/rpmspec.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+> Query a RPM spec file.
+> More information: <https://manned.org/rpmspec>.
+
+- List binary packages which would be generated from the rpm spec file:
+
+`rpmspec --query {{rpm.spec}}`
+
+- Get summary information for single binary packages generated from rpm spec file:
+
+`rpmspec --query --queryformat {{"%{name}: %{summary}\n"}} {{rpm.spec}}`
```
Comment (pixelcmtd) [inline pages/linux/rpmspec.md:12]:
```suggestion
`rpmspec --query --queryformat "{{%{name}: %{summary}\n}}" {{rpm.spec}}`
```

File: pages/linux/rpmspec.md (hunk 1 above)
Comment (marchersimon) [inline pages/linux/rpmspec.md:6]:
```suggestion
- List binary packages which would be generated from a rpm spec file:
```

File: pages/linux/rpmspec.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# rpmspec
+
+> Query a RPM spec file.
+> More information: <https://manned.org/rpmspec>.
+
+- List binary packages which would be generated from the rpm spec file:
+
+`rpmspec --query {{rpm.spec}}`
```
Comment (marchersimon) [inline pages/linux/rpmspec.md:8]:
```suggestion
`rpmspec --query {{path/to/rpm.spec}}`
```
Same for the other examples.

File: pages/linux/rpmspec.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- Get summary information for single binary packages generated from rpm spec file:
```
Comment (marchersimon) [inline pages/linux/rpmspec.md:10]:
```suggestion
- Get summary information for single binary packages generated from a rpm spec file:
```

File: pages/linux/rpmspec.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+`rpmspec --query --queryformat "{{%{name}: %{summary}\n}}" {{rpm.spec}}`
+
+- Get the source package which would be generated from the rpm spec file:
```
Comment (marchersimon) [inline pages/linux/rpmspec.md:14]:
```suggestion
- Get the source package which would be generated from a rpm spec file:
```

File: pages/linux/rpmspec.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+`rpmspec --query --srpm {{rpm.spec}}`
+
+- Parse the rpm spec file to `stdout`:
```
Comment (marchersimon) [inline pages/linux/rpmspec.md:18]:
```suggestion
- Parse a rpm spec file to `stdout`:
```

File: pages/linux/rpmspec.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+> Query a RPM spec file.
+> More information: <https://manned.org/rpmspec>.
+
+- List binary packages which would be generated from a rpm spec file:
+
+`rpmspec --query {{rpm.spec}}`
+
+- Get summary information for single binary packages generated from a rpm spec file:
+
+`rpmspec --query --queryformat "{{%{name}: %{summary}\n}}" {{rpm.spec}}`
```
Comment (mfrw) [inline pages/linux/rpmspec.md:12]:
```suggestion
`rpmspec --query --queryformat "{{%{name}: %{summary}\n}}" {{path/to/rpm.spec}}`
```

File: pages/linux/rpmspec.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+- Get the source package which would be generated from the rpm spec file:
+
+`rpmspec --query --srpm {{rpm.spec}}`
+
+- Parse a rpm spec file to `stdout`:
+
+`rpmspec --parse {{rpm.spec}}`
```
Comment (mfrw) [inline pages/linux/rpmspec.md:20]:
```suggestion
`rpmspec --parse {{path/to/rpm.spec}}`
```

**Review by marchersimon — APPROVED** [review]
Thanks @mfrw

**Pull request merged**